src/ballet/siphash13/fd_siphash13.h

```diff
@@ -0,0 +1,83 @@
+#ifndef HEADER_fd_src_ballet_siphash13_fd_siphash13_h
```

Comment:
It's a nit but can you put an origination notice in the header / impl as well? Someone may not know to look in NOTICE before making changes.